service/storage_proxy.cc

```diff
@@ -3897,7 +3895,8 @@ template<typename Range>
 bool storage_proxy::cannot_hint(const Range& targets, db::write_type type) const {
     // if hints are disabled we "can always hint" since there's going to be no hint generated in this case
     return hints_enabled(type) &&
-            _hints_manager.started() &&
+            _hints_manager.started() && // If the manager hasn't started yet, no mutation will be performed to another node.
+                                        // No hint will need to be stored.
```

Comment:
It's not clear what "performing the mutation" means in this context. In our codebase, "mutation" is usually a verb. You can send it, apply it, etc. I think it's a good occasion to clarify this.

Reply:
I must say that grep on docs/dev doesn't provide a clear definition of what a mutation is either. For example (protocols.md):
```
Scylla's internal protocol is built on top of Seastar's "RPC" messaging
mechanism, itself built on top of TCP. This protocol includes messages of
different types ("verbs") - which are all listed in the source file
`message/messaging_service.cc`.

Here is a non-exhaustive list of things that Scylla does with these messages:

   * "Gossip" between the nodes to maintain knowledge of the cluster topology:
     Which nodes belong to the cluster, their state and token ownership.

   * Read and write (mutation) messages sent from a node receiving a client's
     request (a so-called coordinator) to a replica which holds relevant data.

   * In particular, mutations to special system tables are used to maintain
     cluster-wide agreement on schemas (table definitions).
```